source/change-streams/change-streams.rst

```diff
@@ -679,10 +711,9 @@ It was decided to remove this example from the specification for the following r
 What do the additional error codes mean?
 ----------------------------------------
 
-The `CursorKilled` or `Interrupted` error implies some other actor killed the cursor.
-
-The `CappedPositionLost` error implies falling off of the back of the oplog,
-so resuming is impossible.
+The additional codes were added to match the server changes made in
```

Comment:
Not sure if this section is necessary anymore. I'm open to removing it completely if others agree.

Reply:
I like having it. It does provide convenient rationale for future readers. But also not opposed to removing it, as readers can go through the git history to see this conversation.